skbeam/core/tests/test_correlation.py

```diff
@@ -198,6 +200,28 @@ def test_bad_images():
     assert_array_almost_equal(g2[:, 1], g2_n[:, 1], decimal=3)
 
 
+def test_one_time_from_two_time():
+    num_lev = 1
+    num_buf = 10  # must be even
+    x_dim = 10
+    y_dim = 10
+    stack = 10
+    imgs = np.random.randint(1, 3, (stack, x_dim, y_dim))
+    roi = np.zeros_like(imgs[0])
+    # make sure that the ROIs can be any integers greater than 1.
+    # They do not have to start at 1 and be continuous
+    roi[0:x_dim//10, 0:y_dim//10] = 5
+    roi[x_dim//10:x_dim//5, y_dim//10:y_dim//5] = 3
+
+    two_time = two_time_corr(roi, imgs, stack,
```

Comment:
It is a little difficult to understand exactly what `two_time[0]` means in this test, as you need to understand trace the results of `two_time_corr` through about 3 function calls to understand what exactly it returns.  Having just realized that it is difficult to figure out what `two_time[0]` is, I just put in #407 to update the docs for `two_time_corr`.  Could you also change `two_time` to be `g2, lag_steps, _state` to make the meaning of the return value of `two_time_corr` a little clearer in this test?